### PR TITLE
Base style

### PR DESCRIPTION
**Pull request by gabeborges**

### CODE REVIEW

**Comment by patrickemuller** [comment]
`--amend`, seriously?

**Comment by filipelinhares** [comment]
:-1:

**Comment by ssjr** [comment]
gabrieeeeeel

File: app/assets/stylesheets/atoms/_input.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+input[type="text"] {
+  border: 0;
+  border-radius: 0;
+  width: 100%;
+}
```
Comment (ssjr) [inline app/assets/stylesheets/atoms/_input.scss]: https://robots.thoughtbot.com/no-newline-at-end-of-file

**Comment by andrepinho** [comment]
fuck git history, :shipit:

**Pull request merged**